andela/wg-millennium-falcon
# #157731584 Update muscle list

### What does this PR do?
Updates the list of available muscles as well as the exercises 

### Description of Task to be completed?
New muscles to be added include:
* lateral deltoid - 17
* Obliquus externus abdominis - 18
* Posterious Deltoid - 19
* Forearms - 20

### How should this be manually tested?
After logging on, navigate to the muscles page and view the list of muscles.  There should be 
`lateral deltoid`, `Obliquus externus abdominis`,`Posterious Deltoid` and `Forearms ` muscles with their respective svgs and exercises which were added.

<img width="1093" alt="screen shot 2018-06-13 at 21 28 24" src="https://user-images.githubusercontent.com/26734746/41370772-2f88dcb6-6f51-11e8-9b6d-5218b668ba90.png">

<img width="1102" alt="screen shot 2018-06-13 at 21 30 09" src="https://user-images.githubusercontent.com/26734746/41370801-3c7d75e4-6f51-11e8-969b-6aad8c9e3619.png">

<img width="1118" alt="screen shot 2018-06-13 at 21 30 25" src="https://user-images.githubusercontent.com/26734746/41370813-453536ae-6f51-11e8-845d-3db0026d4d08.png">

<img width="1110" alt="screen shot 2018-06-13 at 21 30 52" src="https://user-images.githubusercontent.com/26734746/41370828-54a35170-6f51-11e8-8ecc-2d6923e6c356.png">


### Any background context you want to provide?
The muscles and their respective exercises were previously not available.

### What are the relevant pivotal tracker stories?

[#157731576](https://www.pivotaltracker.com/story/show/157731576)
